### ISSUE TITLE
By changing batchsize should I change learning rate?

### ISSUE THREAD

**Issue by kpagels** [junyanz/CycleGAN#124]
Hi :)

I have three questions:

1) I rescaled my images to [-1,1] because of the use of tanh in generator, is this not needed? 

2) My images are 140x140. should I change the number of resnet's? And what is the intuition behind this?

2) I changed my batch_size to 8 to speed up and kept the InstanceNormalization. When making this change, should I change the learning rate also? And are there other things I should consider changing? Should I use batchNormalization instead?

Thanks in regards!
